code/game/objects/items/storage/boxes.dm

```diff
@@ -438,6 +438,96 @@
 	for(var/i in 1 to 6)
 		new /obj/item/reagent_containers/food/snacks/donkpocket(src)
 
+/obj/item/storage/box/donkpockets/donkpocketspicy
+	name = "box of spicy-flavoured donk-pockets"
+	desc = "<B>Instructions:</B> <I>Heat in microwave. Product will cool if not eaten within seven minutes.</I>"
+	icon_state = "donkpocketboxspicy"
+	illustration=null
+
+/obj/item/storage/box/donkpockets/donkpocketspicy/ComponentInitialize()
+	. = ..()
+	var/datum/component/storage/STR = GetComponent(/datum/component/storage)
+	STR.set_holdable(list(/obj/item/reagent_containers/food/snacks/donkpocket))

```

Comment:
`/obj/item/storage/box/donkpockets/ComponentInitialize()` already handles the storage. Feel free to remove these lines.

Reply:
Done, thanks for the help!